vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentDetailsComponent.java

```diff
@@ -60,55 +58,79 @@
+  private final ExperimentInformationService experimentInformationService;
   private final HorizontalLayout tagLayout = new HorizontalLayout();
   private final TabSheet experimentSheet = new TabSheet();
-
   private final Div contentExperimentalGroupsTab = new Div();
-
-  private final Board summaryCardBoard = new Board();
-
   private final Div experimentSummary = new Div();
   private final ExperimentalGroupCardCollection experimentalGroupsCollection = new ExperimentalGroupCardCollection();
-  private final CardComponent sampleOriginCard = new CardComponent();
   private final CardComponent blockingVariableCard = new CardComponent();
   private final Button addBlockingVariableButton = new Button("Add");
   private final AddVariablesDialog addVariablesDialog;
   private final AddExperimentalGroupsDialog experimentalGroupsDialog;
-
   private final DisclaimerCard noExperimentalVariablesDefined;
-
   private final CreationCard experimentalGroupCreationCard = CreationCard.create(
-      "Add Experimental Group");
+      "Add more experimental groups");
 
 
   public ExperimentDetailsComponent(@Autowired ProjectInformationService projectInformationService,
       @Autowired ExperimentInformationService experimentInformationService) {
     Objects.requireNonNull(projectInformationService);
-    Objects.requireNonNull(experimentInformationService);
+    this.experimentInformationService = Objects.requireNonNull(experimentInformationService);
+
     this.addVariablesDialog = new AddVariablesDialog(experimentInformationService);
     this.noExperimentalVariablesDefined = createDisclaimer();
-    getContent().indentContent(false);
-    initTagAndNotesLayout();
-    initTabSheet(experimentInformationService);
-    addCreationCard();
-    experimentalGroupsDialog = createExperimentalGroupDialog();
+    this.experimentalGroupsDialog = createExperimentalGroupDialog();
 
-    this.handler = new Handler(experimentInformationService);
-    setUpCreationCard();
+    layoutComponent();
+    configureComponent();
   }
 
-  public DisclaimerCard createDisclaimer() {
+  private DisclaimerCard createDisclaimer() {
     var disclaimer = DisclaimerCard.createWithTitle("Missing variables",
         "No experiment variables defined", "Add");
     disclaimer.subscribe(this::handleEvent);
     return disclaimer;
   }
 
+  private AddExperimentalGroupsDialog createExperimentalGroupDialog() {
+    AddExperimentalGroupsDialog dialog = new AddExperimentalGroupsDialog();
+    dialog.addExperimentalGroupSubmitListener(
+        groupSubmitted -> onGroupSubmitted(groupSubmitted));
+    return dialog;
+  }
+
+  private void layoutComponent() {
+    initTagAndNotesLayout();
+    layoutTabSheet();
+  }
+
+  private void configureComponent() {
+    configureExperimentalGroupCreation();
+    addCloseListenerForAddVariableDialog();
+  }
+
+  private void handleEvent(DisclaimerConfirmedEvent disclaimerConfirmedEvent) {
+    experimentSheet.setSelectedIndex(0);
```

Comment:
NIT: 
You could think about defining the Tabs explicitly like e.g. experimentSummaryTab, experimentalgroupTab and use the setSelected(Tab tab) method instead of relying on the index